sdk/program/src/last_restart_slot.rs

```diff
@@ -0,0 +1,12 @@
+//! Information about the last restart slot (hard fork).
+
+use solana_sdk_macro::CloneZeroed;
+
+pub type Slot = u64;
```

Comment:
`Slot` is already defined in the clock mod:
```suggestion
use crate::clock::Slot;
```